src/main/java/cart/auth/AuthArgumentResolver.java

```diff
@@ -0,0 +1,29 @@
+package cart.auth;
+
+import org.springframework.core.MethodParameter;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class AuthArgumentResolver implements HandlerMethodArgumentResolver {
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(AuthResolving.class);
+    }
+
+    @Override
+    public Object resolveArgument(
```

Comment:
```suggestion
    public AuthInfo resolveArgument(
```
AuthInfo로 바꿔도 되지 않나요? 
체크해봐주세요!

Reply:
밑에서 언급해주신 throws Exception 부분도 그렇고 인터페이스를 구현하는 과정에서 메서드 시그니처를 반드시 그대로 따를 필요는 없다는 것은 몰랐네요... 아직 스프링 프레임워크의 전체적인 구조를 머리속에 그리지 못하다보니 이런 세부적인 것들을 건드리기가 무섭더라고요... ㅎㅎ 감사합니다!